public/director-upload-movie.php

```diff
@@ -1,10 +1,85 @@
+
+
+
+
 <!DOCTYPE html>
 <html lang="en">
 
   <head>
     <title>template</title>
     <link rel="stylesheet" href="./assets/css/styleupload.css">
-    <?php include "../templates/head.php" ?>
+    <?php include "../templates/head.php";
+          require("../utils/movies.php");
+          //UserId needs to be filtered still
+          $userId = 2;
+          if(!empty($_POST['MovieTitle'])){
+            $title = $_POST['MovieTitle'];
+          }
+
+          if(!empty($_POST['Category'])){
+            $genre = $_POST['Category'];
+          }
+          
+          if(!empty($_POST['AgeRating'])){
+            $ageRating = $_POST['AgeRating'];
+          }
+          
+          if(!empty($_POST['filmGuide'])){
+            $filmGuide = $_POST['filmGuide'];
+          }
+
+          if(!empty($_POST['Description'])){
+            $description = $_POST['Description'];
+          }
+
+          if(isset($_POST['upload'])){
+            if(!empty($_POST['MovieTitle']) AND !empty($_POST['Category']) AND !empty($_POST['AgeRating']) AND !empty($_POST['filmGuide'])  AND !empty($_FILES['Movie']) AND !empty($_FILES['Thumbnail']) AND !empty($_POST['Description'])){
+                $allowed = array('mp4', 'jpg');
```

Comment:
De extensies zou je even moeten halen src/constants.php. Hier staan extensies in voor video's en afbeeldingen. Ook de MIME types staan hierin